indico/modules/users/schemas.py

```diff
@@ -83,3 +106,360 @@ class Meta:
     location = fields.String(attribute='event.location.venue_name')
     chain_titles = fields.List(fields.String(), attribute='category.chain_titles')
     label_markup = fields.Function(lambda e: e.get_label_markup('mini'))
+
+
+# Schemas for user data export
+
+class DataExportRequestSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = DataExportRequest
+        fields = ('state', 'requested_dt', 'selected_options', 'max_size_exceeded', 'url')
+
+    selected_options = List(Enum(DataExportOptions))
+
+
+class PersonalTokenExportSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = PersonalToken
+        fields = ('name', 'revoked_dt', 'created_dt', 'last_used_dt', 'last_used_ip', 'use_count', 'scopes')
+
+    scopes = Function(lambda token: sorted(list(token.scopes)))
+
+
+class OAuthApplicationExportSchema(mm.SQLAlchemyAutoSchema):
+    class Meta:
+        model = OAuthApplication
+        fields = ('name', 'description', 'allowed_scopes', 'redirect_uris', 'is_enabled', 'is_trusted',
+                  'allow_pkce_flow')
```

Comment:
I don't think anything besides name/description should be exported here - those are system-level apps, and things like the list of redirect URIs are none of the user's business